sample/ApiSamples/Program.cs

```diff
@@ -132,5 +133,37 @@ public static void SampleError()
 					s.Tags["fooSpan"] = "barSpan";
 				});
 			});
+
+
+		//1 transaction with 2 spans
+		//1 transaction with 1 span that has a sub span
+		public static void TwoTransactionWith2Spans()
+		{
+
+			//1 transaction 2 spans (both have the transaction as parent)
+			Agent.Tracer.CaptureTransaction("TestTransaction1", "TestType1",
+				t =>
+				{
+					t.CaptureSpan("TestSpan", "TestSpanName", s =>
+					{
+						Thread.Sleep(20);
```

Comment:
It's not really a problem to have zero duration, but I'd prefer to avoid it.

So I have 2 reasons for doing this: 
-It's just that I don't want to have a span that has 0 duration. These are live samples... you can start them and they show up in kibana (e.g. the screenshot from the issue description is also from this `ApiSamples` app. So having spans with 0 duration would ruin the demo-ability of this code.
-This is also kind of serves as sample code, and I think people can easily interpret `Thread.Sleep` as any arbitrary work. Showing empty bodies would be in my opinion harder to read for people.